### PR TITLE
Indicated Arch Linux(arch) supported after verifying basic

### PR DESCRIPTION
**Pull request by riddlemethrice**
functionality in 'systemd' 2.0.0 cookbook against Chef 12.9.

### CODE REVIEW

**Comment by ranjib** [comment]
@riddlemethrice you'll help us with testing this on arch ? I am not using it personally, i run ubuntu mostly , across x86, i386 & armhf. I think @nathwill runs fedora/rel derivatives.

I think we can skip the rubocop formatting fixes and version changes as part of this PR. 

:+1:

**Comment by riddlemethrice** [comment]
G'morning, Ranjib! Yes, I'm actively using this cookbook in Arch on
arm(Raspberry Pi) and x86. I'll help in what ever way I can. =)

**Comment by ranjib** [comment]
that is so awesome :-) . Im running a pi cluster on 16.04, and with chef+systemd+ influxdb/grafana/telegraf stuff to manage whole bunch of things,

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 2.1.1 / 2016-05-18
+
+* RuboCop whitespace issues fixed
+
+# 2.1.0 / 2016-05-18
```
Comment (nathwill) [inline CHANGELOG.md:1]: no need to version twice here, we haven't merged/released it yet :)

**Comment by nathwill** [comment]
+1 for @ranjib's comments, let's skip the rubocop fixes for this PR and just handle the extra supports addition (can even skip the version bump if you like).

**Comment by riddlemethrice** [comment]
Thanks for the feedback and I think I've made the changes necessary but let me know if not. =)

**Comment by nathwill** [comment]
:+1: thanks @riddlemethrice !

**Pull request merged**